Autopilot/Src/altimeterThreadInterface.cpp

```diff
@@ -0,0 +1,12 @@
+#include "altimeterThreadInterface.h"
+#include "imu.hpp"
```

Comment:
To be an imu is to be an altimeter. So, to move is to be high. Ergo, exercise is a drug :))

(I am going to assume you will replace this with `airspeed.hpp` when it is ready. But please add a comment regardless :))

Also, add your name to all these files! Get that author cred!